shared/bash_remediation_functions/dconf_lock.sh

```diff
@@ -0,0 +1,26 @@
+# Function to configure DConf locks for RHEL and Fedora systems.
+#
+# Example Call(s):
+#
+#     dconf_lock 'org/gnome/login-screen' 'banner-message-enable' 'local.d' 'banner'
+#
```

Comment:
It is not clear what this function is supposed to do, so I can't verify it's functionality.

Reply:
@matejak this function adds an entry to `/etc/dconf/db/${db}/locks/${file}` which prevents a user from changing the setting.